crowdin.yml

```diff
@@ -1,5 +1,5 @@
 commit_message: New translations %language% from Crowdin
-append_commit_message: 
+append_commit_message:
 files:
   - source: locale/crowdin.ts
-    translation: /locale/%locale_with_underscore%.%file_extension%
+    translation: /locale/%two_letters_code%.%file_extension%
```

Comment:
what about es_AR  es_BO language?

Reply:
We can override them here.

![image](https://github.com/user-attachments/assets/86b458ee-5f58-4c04-b42a-c07ba6827e99)